### ISSUE TITLE
Best pre-trained weights for HRNet + OCR

### ISSUE THREAD

**Issue by netw0rkf10w** [openseg-group/openseg.pytorch#17]
Hello,

Thanks for making the code and the pre-trained models available!

I would like to know to reproduce your results on the Cityscapes test set (mIoU 84.5/84.2 with/without SegFix) from your provided pre-trained model.

Should I take the 80000-iteration OCR HRNet-W48 weights that you listed in the [model zoo](https://github.com/openseg-group/openseg.pytorch/blob/master/MODEL_ZOO.md)?

Thank you in advance for your response.

**Comment by PkuRainBow** [comment]
@netw0rkf10w We would like to release the complete training scripts to reproduce the best performance soon. Please being patient.

**Comment by PkuRainBow** [comment]
@netw0rkf10w  Please check the overall pipeline here: https://github.com/openseg-group/openseg.pytorch/blob/master/MODEL_ZOO.md#how-to-reproduce-the-hrnet--ocr-with-mapillary-pretraining and we will upload the checkpoints latter.
